.vscode/launch.json

```diff
@@ -37,7 +37,8 @@
         "--nolazy"
       ],
       "env": {
-        "NODE_ENV": "development"
+        "NODE_ENV": "development",
+        "TEST_FILTER": "^jsdoc_types$"
```

Comment:
I see your point but I worry that it's really hard to notice -- the test runner prints all of the file names regardless of whether it runs the tests or not.